src/content/1.2/code/ocaml/snippet06.ml

```diff
@@ -1,3 +1,2 @@
 type void
```

Comment:
```ocaml
type void = |
```
or just use the word `void` directly, it's not like the haskell snippet has `import Data.Void (Void, absurd)`